packages/components/src/styles/common-styles.ts

```diff
@@ -5,18 +5,51 @@ import type { ThemedColors } from './';
 import {
   borderWidthBase,
   frostedGlassStyle,
+  motionDurationModerate,
+  motionDurationShort,
+  motionDurationLong,
+  motionDurationVeryLong,
+  motionEasingBase,
+  motionEasingIn,
+  motionEasingOut,
   themeDarkBackgroundShading,
   themeLightBackgroundShading,
   themeLightStateFocus,
 } from '@porsche-design-system/utilities-v2';
 import { getThemedColors, prefersColorSchemeDarkMediaQuery } from './';
 import { isThemeDark } from '../utils';
+import type * as fromMotionType from '@porsche-design-system/utilities-v2/dist/esm/motion';
+type WithoutMotionDurationPrefix<T> = T extends `motionDuration${infer P}` ? Uncapitalize<P> : never;
+export type MotionDurationKey = WithoutMotionDurationPrefix<keyof typeof fromMotionType>;
+type WithoutMotionEasingPrefix<T> = T extends `motionEasing${infer P}` ? Uncapitalize<P> : never;
+export type MotionEasingKey = WithoutMotionEasingPrefix<keyof typeof fromMotionType>;
 
-export const transitionDuration = 'var(--p-transition-duration, .24s)';
-const transitionTimingFunction = 'ease';
+const motionDurationMap: Record<MotionDurationKey, string> = {
+  short: motionDurationShort,
+  moderate: motionDurationModerate,
+  long: motionDurationLong,
+  veryLong: motionDurationVeryLong,
+};
+
+export const motionEasingMap: Record<MotionEasingKey | 'linear', string> = {
+  base: motionEasingBase,
+  in: motionEasingIn,
+  out: motionEasingOut,
+  linear: 'linear',
+};
+
+export const cssVariableTransitionDuration = '--p-transition-duration';
+export const cssVariableAnimationDuration = '--p-animation-duration';
 
-export const getTransition = (cssProperty: keyof PropertiesHyphen): string =>
-  `${cssProperty} ${transitionDuration} ${transitionTimingFunction}`;
+export const getTransition = (
+  cssProperty: keyof PropertiesHyphen,
+  duration: MotionDurationKey = 'short',
+  easing: keyof typeof motionEasingMap | 'none' = 'base',
+  delay?: MotionDurationKey
+): string =>
+  `${cssProperty} var(${cssVariableTransitionDuration}, ${motionDurationMap[duration]})${
+    easing === 'none' ? '' : ` ${motionEasingMap[easing]}`
```

Comment:
I think easing none will result in it's not defined for the browser which then takes `ease` as default, right? Isn't this then misleading value naming?